actions/form/class.Instance.php

```diff
@@ -251,4 +250,11 @@ private function isList(core_kernel_classes_Property $property): bool
             new core_kernel_classes_Class(TaoOntology::CLASS_URI_LIST)
         );
     }
+
+    private function isEmptyLabel($element): bool
+    {
+        return $element instanceof tao_helpers_form_elements_Label
+            && (($element->getRawValue() === null)
+                || (strlen($element->getRawValue()) === 0));
```

Comment:
You can achieve same result with:

```suggestion
        return $element instanceof tao_helpers_form_elements_Label
            && empty($element->getRawValue());
```

Reply:
Changed, thanks! :)